tests/Functional/Driver/PDO/SQLSrv/DriverTest.php

```diff
@@ -77,4 +78,31 @@ public function testDriverOptions(): void
                 ->getAttribute(PDO::ATTR_CASE)
         );
     }
+
+    /**
+     * @dataProvider stringDataProvider
+     */
+    public function testDriverStringBinding(string $bindingType, string $expectedSqlType): void
```

Comment:
I'd rather create a new functional `StringBindingTest` that would run with all drivers that support ASCII binding (`sqlsrv` and `pdo_sqlsrv`).